pom.xml

```diff
@@ -1298,6 +1299,18 @@
         <artifactId>json-path</artifactId>
         <version>${json-path.version}</version>
```

Comment:
Instead of upgrading json-smart, I was wondering if we could drop json-path completely? Have you explored this option?

Reply:
The dep comes from Calcite and in the context of Hive I get the impression that we don't use that code path so maybe we can get rid of this dep especially since json-path does not seem to be actively maintained.

Reply:
The build will probably pass but try also to run some tests to see that they don't fail.